langchain/src/tools/index.ts

```diff
@@ -47,3 +47,10 @@ export {
   formatToOpenAIFunction,
   formatToOpenAITool,
 } from "./convert_to_openai.js";
+
+export { OutlookSendMailTool, OutlookReadMailTool } from "./outlook/index.js";
+export {
+  GooglePlacesAPI,
+  type GooglePlacesAPIParams,
+} from "./google_places.js";
```

Comment:
Remove from `index` entrypoint - can't have things that rely on optional deps here